omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -78,22 +219,24 @@
                     regimenStartedDate = regimenBeforeDTGStartedDate;
                 }
 
-                disable_fields("duration-on-current-regimen");
-                disable_fields("duration-on-art");
+                disable_fields("99025");
```

Comment:
@slubwama why are you replacing the IDs with concept codes - the ids were added to make the referencing clearer and easier to use

Reply:
done